### PR TITLE
Run tests against PHP 8.3

### PR DESCRIPTION
**Pull request by n7studios**
## Summary

Run tests against PHP 8.3

## Testing

Existing tests pass.

## Checklist

* [x] I have [written a test](https://github.com/ConvertKit/convertkit-wordpress/blob/main/TESTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](https://github.com/ConvertKit/convertkit-wordpress/blob/main/TESTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](https://github.com/ConvertKit/convertkit-wordpress/blob/main/TESTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](https://github.com/ConvertKit/convertkit-wordpress/blob/main/DEVELOPMENT.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](https://github.com/ConvertKit/convertkit-wordpress/blob/main/DEVELOPMENT.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
     strategy:
       matrix:
         wp-versions: [ 'latest' ] #[ 'latest', '6.1.1' ]
-        php-versions: [ '7.4', '8.0', '8.1', '8.2' ] #[ '7.4', '8.0', '8.1', '8.2' ]
+        php-versions: [ '7.4', '8.0', '8.1', '8.2', '8.3' ] #[ '7.4', '8.0', '8.1', '8.2' ]
```
Comment (noelherrick) [inline .github/workflows/tests.yml:45]: Why are we deleting the ngnix config for 7.4, 8.0, and 8.1 if we're still testing against them?

Comment (n7studios) [reply .github/workflows/tests.yml:45]: nginx is never used in the GitHub action for running unit tests, so the nginx config files aren't needed here.

Comment (noelherrick) [reply .github/workflows/tests.yml:45]: Got it, what are they for then?

**Review by noelherrick — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (n7studios) [reply .github/workflows/tests.yml:45]: The `php-versions` are used in the testing matrix to run unit tests against each PHP version (example results can be seen [here](https://github.com/ConvertKit/convertkit-wordpress-libraries/actions/runs/7103955903)).

nginx configuration files remain in the Plugin repositories, as those GitHub actions run acceptance tests through the browser, interacting with WordPress and a Plugin (e.g. adding a ConvertKit Form block). nginx acts as the web server for running WordPress for those tests.

**Pull request merged**